### ISSUE TITLE
How can I add a swipe hint?

### ISSUE THREAD

**Issue by oferRounds** [zhxnlai/ZLSwipeableViewSwift#57]
I'm using the the class in my on boarding.
I'd like to add a delicate card move to hint the user that the view is he's currently seeing is swipeable.

Anyone has an idea how can I achieve this?
I'd like the change to appear "real", as if someone really did a little swipe to the right, and then removed his finger.

**Comment by AndrewSB** [comment]
@oferRounds you can try some programmatic touches. "Drag" some to the right, wait half a second, let go? You should be able to pass an x offset into the `ZLSwipeableView`'s gesture recognizer

**Comment by oferRounds** [comment]
Hi @AndrewSB 
Thanks for replying!

I'm actually using the old obj-c `ZLSwipeableView's`, and I have tried something similar:

Passing an array of `movement` values (which I recorded earlier when testing a slight hint) to a method which I added which is similar to the gesture handler method. This method handles these values (the first value is treated as the `began`, the last as the `end` and all the others - as `changed`), but it didn't really work - the top card did not move...

**Comment by AndrewSB** [comment]
Hmm, I see. What was the reason for switching back to the obj-c library?
What about it didn't work? Did the card move at all? Ths would be a cool
feature to add as an example to the demo app for the project. I'd love to
work with you to change the library/ figure out a good implementation that
other people will be able to learn from

**Comment by oferRounds** [comment]
Hi @AndrewSB 
First - I really apologise for the delay!

Regarding your first question - I actually never used the Swift version of `ZLSwipeableView`. I started to write my application when Swift was in its early stages, and the the `ZLSwipeableViewSwift`was not yet available. Since then, I have not migrated to it because my controller, the one which uses `ZLSwipeableView`, is written in obj-c.

I posted my question also here, since I figured out there are similarities between the libraries, so it would be possible for me to implement it also in the obj-c library.

Thanks for offering to work on it together! Although it sounds appealing to me, I won't be able to do it at the current time, mostly because I'm not yet familiar enough with Swift. Also, at this stage I would be more interested to add it to the obj-c library. 
Sorry for that - I do appreciate your offer very much!

As for your second question - no, it did not move at all...
